podcasts/New Search/Views/Results/PredictiveList.swift

```diff
@@ -9,33 +9,74 @@ struct PredictiveList: View {
     @EnvironmentObject var searchResults: SearchResultsModel
     @EnvironmentObject var searchHistory: SearchHistoryModel
 
-    var body: some View {
-        ForEach(searchResults.predictive, id: \.self) { predictiveSearch in
-            switch predictiveSearch.type {
-                case .term(let searchTerm):
-                    VStack {
-                        termRow(term: searchTerm)
-                        if !FeatureFlag.searchImprovements.enabled {
-                            ThemedDivider()
-                        }
+    var termsResults: [PredictiveSearchResult] {
+        searchResults.predictive.filter { result in
+            if case .term = result.type {
+                return true
+            } else {
+                return false
+            }
```

Comment:
Maybe just return false directly with no else?
```suggestion
            }
            return false
```

Reply:
Updated here: https://github.com/Automattic/pocket-casts-ios/pull/3587/commits/9e4b2a2a99122ac74fe2b39584c45e4839074e7d